IanGRap/Squish
# Lobby ui animation bugfix

Contributors: Alfredo

#315 Fix issue where play button was not properly highlighting after backing out of certain lobby UI menus.

Changes: 
* LobbyUIScreenManager.cs: got rid of the _previouslySelectedButton variable that was not being used anywhere else. This was causing SetSelected() to be called twice.

How to test: 
* Run through the game once to make sure the merge didn't break anything. 
* Make sure that whenever you go back to the Main menu the play button highlights correctly
